pkg/providers/rocketchat/rocketchat.go

```
@@ -0,0 +1,145 @@
+
+func send(webhookURL string, payload WebhookPayload) error {
+	jsonPayload, err := json.Marshal(payload)
+	if err != nil {
+		return errors.Wrap(err, "failed to marshal rocketchat payload")
+	}
+
+	resp, err := http.Post(webhookURL, "application/json", bytes.NewBuffer(jsonPayload))
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)